packages/orchestration/src/orchestration-types.ts

```diff
@@ -315,3 +320,34 @@ export interface TranslationConfigParams {
    */
   targetLanguage: string;
 }
+
+/**
+ * Filter configuration for Azure Content Safety.
+ */
+export type AzureContentSafety = AzureContentSafetyOutput;
```

Comment:
```suggestion
export type AzureContentSafety = AzureContentSafetyInput;
```

[req] The old types should reflect the input filtering config so user can still use the `build` function for input filtering config as well. Adding a new property is not a breaking change. In the createCompletionPostRequest method, we should remove PromptShield if it was added there (But I believe there is a separate ticket handle prompt shield properly)

Reply:
`AzureContentSafety` is used in the deprecated `buildAzureContentFilter()` function and we export this type as public. If I change it to input, it will have an additional field. User will see this field in autocompletion, which does nothing. I think this should be properly handled in the upcoming `PromptShield` BLI.

Reply:
> I think this should be properly handled in the upcoming PromptShield BLI.

Yeah makes sense then.

Reply:
Yeah I agree, I wanted to avoid implementing anything for PromptShield already in this PR. I try to make no changes to our client just for now.